src/uct/ib/base/ib_iface.c

```diff
@@ -56,6 +56,10 @@ ucs_config_field_t uct_ib_iface_config_table[] = {
   {"", "", NULL,
    ucs_offsetof(uct_ib_iface_config_t, super), UCS_CONFIG_TYPE_TABLE(uct_iface_config_table)},
 
+  {"SEG_SIZE", "8192",
```

Comment:
how does it correlate with UCX_SEG_SIZE, used for pre-registered buffers in AM/UCP? If I set UCX_SEG_SIZE to some value, will it affect this var as well?

Reply:
@dmitrygladkov, the SEG_SIZE also used for BCOPY, right ? If so, I suggest update documentation to reflect this more explicitly. SEG_SIZE is used to XXX protocols to define XXX.

Reply:
@shamisp Thanks for the comment! This is fixed now